### DISCUSSION TITLE
Help inverting a key

### DISCUSSION THREAD

**Discussion by JesseCSlater** [sezanzeb/input-remapper#660] [General]
I want to invert my j key, so that while I am not pressing it, it is held down, and when I press it, it is released. If that sounds like a strange request, its for a video game. Ideally I would also have another key which toggles the inversion on and off, so that I could stop holding down the key when I was out of game. That is not as important though, because I am willing to just disable input-remapper whenever I am not in game.
Is that possible with input-remapper? I was messing around with macros, but I did not figure out a way to do it.

**Comment by jonasBoss** [comment]
try this:
```
key_up(j)
.hold()
.key_down(j)
```

**Reply by Zliquat** [reply]
Amazing work around. You've saved me 3 years later.

I'm in a bit of disbelief there is no toggle to invert inputs when some of the racing wheels listed as supported have their analog axis' inverted and have to be toggled as such in game.

I've spent hours trying to find a way to invert digital inputs because Forza doesn't support dual controller inputs / mouse bindings
